packages/angular/projects/angular/src/components/radio-button/radio-button.component.ts

```diff
@@ -62,15 +73,28 @@ implements OnDestroy {
 
   @Input() label?: string;
 
+  // TODO: Add this template to core prop types.
+  @Input() labelTemplate: AngularTemplate<LabelViewComponent> = LabelViewComponent;
+
+  // TODO: Add this template to core prop types.
+  @Input() radioTemplate: AngularTemplate<RadioViewComponent> = RadioViewComponent;
+
   @Output() onChange = new EventEmitter<Event>();
 
   @Output() onClick = new EventEmitter<MouseEvent>();
 
   checked$ = this.strategy.checked$;
 
```

Comment:
Suggestion: May be we can return boolean from this.strategy.checked$ and remove map here?

Reply:
We already return a boolean as stream in ``` this.strategy.checked$ ``` ( ```  Observable<boolean> ```)
We need to wrap ```checked``` in the object for the ```dx-dynamic-template``` component.